walnut/converter/dump.json

```diff
@@ -0,0 +1,364 @@
+[INFO] Using config file 'json:///home/alidock/.sw/slc8_x86-64/QualityControl/master-1/etc/basic.json'
+2020-06-04 16:59:04.933371     QC infologger initialized
+2020-06-04 16:59:04.933395     QC version 0.26.1
+2020-06-04 16:59:04.933554     policyName : tst-raw
+2020-06-04 16:59:04.933588     Configuration loaded : 
+2020-06-04 16:59:04.933595     >> Task name : QcTask
+2020-06-04 16:59:04.933599     >> Module name : QcSkeleton
+2020-06-04 16:59:04.933604     >> Detector name : TST
+2020-06-04 16:59:04.933614     >> Cycle duration seconds : 10
+2020-06-04 16:59:04.933619     >> Max number cycles : -1
+2020-06-04 16:59:04.933736     >> Check name : QcCheck
+2020-06-04 16:59:04.933843     >> Inputs (1): QC_QcTask-mo_0  checks (1): QcCheck  stores (1): QC_QcTask-mo_0 
+{
+    "workflow": [
+        {
+            "name": "producer-0",
+            "inputs": [],
+            "outputs": [
+                {
+                    "binding": "out",
+                    "origin": "TST",
+                    "description": "RAWDATA",
+                    "subspec": 0,
+                    "lifetime": 0
+                }
+            ],
+            "options": [],
+            "rank": 0,
+            "nSlots": 1,
+            "inputTimeSliceId": 0,
+            "maxInputTimeslices": 1
```

Comment:
Not sure if it's a mistake but `maxInputTimeSlices` would be a better name, don't you think?

Reply:
It comes from DPL. Your suggestion might indeed be more consistent...
Pinging @ktf in case he'd like to fix it... or confirm whether it's formatted like this on purpose.

Reply:
Nah, just a typo I guess. PR welcome. ;-)